### DISCUSSION TITLE
Noob - Macbook Air

### DISCUSSION THREAD

**Discussion by Antonie1962** [freeplane/freeplane#312] [Help] [answered]
Howdee, it seems the app is only supported to run on OS X, not on macOS.

What would be required to run the app on the Macbook Air?

Your help will appreciated
Regards
Tom

**Comment by dpolivaev** [comment, marked as answer]
Read https://sourceforge.net/p/freeplane/discussion/758437/thread/e4374ed1c2/ and run it
